src/util/subprocess.h

```diff
@@ -904,8 +904,6 @@ class Streams
  * 3. wait()             - Wait for the child to exit.
  * 4. retcode()          - The return code of the exited child.
  * 5. pid()              - PID of the spawned child.
- * 6. poll()             - Check the status of the running child.
- * 7. kill(sig_num)      - Kill the child. SIGTERM used by default.
  * 8. send(...)          - Send input to the input channel of the child.
```

Comment:
The numbering of this list is off after this change. That might be confusing to someone in the future. I would suggest either updating the following numbering or something like.

```
 * 6. (deleted)
 * 7. (deleted)
```

Reply:
Ok, strange, I guess I was looking at an outdated version of the code or something.